config/default.yml

```diff
@@ -23,6 +23,9 @@ RSpec/ExampleWording:
     not: does not
   IgnoredWords: []
 
+RSpec/BeforeAfterAll:
+  Description: 'Avoid the use of before/after(:all) as they are not rolled back and leads to flakiness.'
```

Comment:
Agree. Sorry for the feedback loop I should have been more careful about the description. Thanks for taking the time to review and suggest improvements... ❤️